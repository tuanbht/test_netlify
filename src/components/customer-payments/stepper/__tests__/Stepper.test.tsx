import { shallow } from 'enzyme';
import React from 'react';
import Stepper from '../index';

describe('Stepper', () => {
  it('renders template correctly', () => {
    const steps = ['step-1', 'step-2', 'step-3'];

    expect(
      shallow(
        <Stepper activeStep={0} stepper={steps}>
          <div>Children</div>
        </Stepper>,
      ),
    ).toMatchSnapshot();
  });
});
