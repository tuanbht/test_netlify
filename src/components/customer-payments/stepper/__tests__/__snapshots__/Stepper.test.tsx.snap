// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Stepper renders template correctly 1`] = `
<div
  className="makeStyles-root-1"
>
  <WithStyles(ForwardRef(Grid))
    container={true}
    justify="space-between"
  >
    <WithStyles(ForwardRef(Grid))
      item={true}
      key="step-1"
    >
      <span
        className="makeStyles-label-2 makeStyles-activeLabel-3"
      >
        step-1
      </span>
    </WithStyles(ForwardRef(Grid))>
    <WithStyles(ForwardRef(Grid))
      item={true}
      key="step-2"
    >
      <span
        className="makeStyles-label-2"
      >
        step-2
      </span>
    </WithStyles(ForwardRef(Grid))>
    <WithStyles(ForwardRef(Grid))
      item={true}
      key="step-3"
    >
      <span
        className="makeStyles-label-2"
      >
        step-3
      </span>
    </WithStyles(ForwardRef(Grid))>
  </WithStyles(ForwardRef(Grid))>
  <WithStyles(WithStyles(ForwardRef(LinearProgress)))
    className="makeStyles-linear-4"
    value={33.333333333333336}
    variant="determinate"
  />
  <div
    className="makeStyles-activeStep-5"
  >
    <div>
      Children
    </div>
  </div>
</div>
`;
